<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1273</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="sale_list">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>451</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>All avaliable sales</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saleDocx">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export info about all sales *.docx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saleExcel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export info about all sales *.excel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search sale by date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchSaleBtn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QListView" name="search_sale_list">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>100</y>
      <width>351</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add new sale</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="seller_id">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter seller`s id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="products">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>80</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter products splited by coma</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="amounts">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>120</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter amounts splited by coma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSale">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="choseDate">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="idEnter">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>210</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>sale id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updSales">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Update list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delSale">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>240</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QListView" name="allSellers">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>451</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>All sellers</string>
    </property>
   </widget>
   <widget class="QListView" name="allProds">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>361</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>All products</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1273</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
